helm/superset/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.serviceAccount.create -}}
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: v1
```

Comment:
Should Superset chart create Service Account? I suggest we simplify and assume that a user will create a SA with needed permissions. It may be done in several ways. For example it's more common to create IRSA in AWS with Terraform.